docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -0,0 +1,411 @@
+----
+
+.Procedure
+
+This procedure assumes that some of the steps are run on the source node that
+we want to decommission, while other steps are run on the target node that is
+supposed to host the redeployed daemon.
+
+. Before start the migration, ssh into the target node and enable the firewall
+rules required to reach a Mon service:
++
+----
+for port in 3300 6789; {
+    ssh heat-admin@<target_node> sudo iptables -I INPUT \
+    -p tcp -m tcp --dport $port -m conntrack --ctstate NEW \
+    -j ACCEPT;
+}
+----
+
+- Replace <target_node> with the hostname of the node that is supposed to host
+the new mon
+
+. Check the rules are properly applied and persist them:
++
+----
+sudo iptables-save
+sudo systemctl restart iptables
+----
+
+. To migrate the existing Mons to the target {Ceph} nodes, the first step is to
+create the following {Ceph} spec from the first mon (or the first controller)
+and modify the placement based on the appropriate label.
++
+[source,yaml]
+----
+service_type: mon
+service_id: mon
+placement:
+  label: mon
+----
+
+. Save the spec in /tmp/mon.yaml
+. Apply the spec with cephadm using the orchestrator:
++
+----
+sudo cephadm shell -m /tmp/mon.yaml
+ceph orch apply -i /mnt/mon.yaml
+----
++
+[NOTE]
+The effect of applying the `mon.yaml` spec is to normalize the existng placement
+strategy to use `labels` instead of `hosts`. By doing this any node with the `mon`
+label is able to host a Ceph mon daemon.
+The step above can be executed once, and it shouldn't be repeated in case of
+multiple iterations over this procedure when multiple Ceph Mons are migrated.
+
+. Before moving to the next step, check the status of the {CephCluster} and the
+orchestrator daemons list: make sure the three mons are in quorum and listed by
+the `ceph orch` command:
++
+----
+# ceph -s
+  cluster:
+    id:     f6ec3ebe-26f7-56c8-985d-eb974e8e08e3
+    health: HEALTH_OK
+
+  services:
+    mon: 3 daemons, quorum oc0-controller-0,oc0-controller-1,oc0-controller-2 (age 19m)
+    mgr: oc0-controller-0.xzgtvo(active, since 32m), standbys: oc0-controller-1.mtxohd, oc0-controller-2.ahrgsk
+    osd: 8 osds: 8 up (since 12m), 8 in (since 18m); 1 remapped pgs
+
+  data:
+    pools:   1 pools, 1 pgs
+    objects: 0 objects, 0 B
+    usage:   43 MiB used, 400 GiB / 400 GiB avail
+    pgs:     1 active+clean
+----
++
+----
+[ceph: root@oc0-controller-0 /]# ceph orch host ls
+HOST              ADDR           LABELS          STATUS
+oc0-ceph-0        192.168.24.14  osd
+oc0-ceph-1        192.168.24.7   osd
+oc0-controller-0  192.168.24.15  _admin mgr mon
+oc0-controller-1  192.168.24.23  _admin mgr mon
+oc0-controller-2  192.168.24.13  _admin mgr mon
+----
+
+. On the source node, backup the `/etc/ceph/` directory: this allows, in case
+of issues, to have the ability to execute cephadm and get a shell to the ceph
+cluster from the source node:
```

Comment:
```suggestion
. On the source node, back up the `/etc/ceph/` directory. In case of issues, this backup allows you to have the ability to execute cephadm and get a shell to the {Ceph} cluster from the source node:
```

Reply:
Done